driver/csiplugin/controllerserver.go

```diff
@@ -1081,10 +1082,11 @@ func (cs *ScaleControllerServer) setScaleVolume(ctx context.Context, req *csi.Cr
 	}
 
 	scaleVol.VolName = volName
+	/* #nosec G115 -- false positive */
 	if scaleVol.IsFilesetBased && uint64(volSize) < smallestVolSize {
```

Comment:
Check this once. seems this nosec should be next line.

Reply:
Same as above